Make.inc

```diff
@@ -481,10 +486,13 @@ OBJCOPY := $(CROSS_COMPILE)objcopy
 # file extensions
 ifeq ($(OS), WINNT)
   SHLIB_EXT := dll
+  STATICLIB_EXT := lib
```

Comment:
Also under mingw64? ce33683 changed that we are now linking sys.so statically against compiler-rt.